ui/round/src/clock/ctrl.js

```diff
@@ -21,20 +21,24 @@ module.exports = function(data, opts) {
 
   var times;
 
-  function update(white, black) {
+  function update(white, black, delay) {
+    if (delay === undefined) delay = 0;
     times = {
       white: white * 1000,
       black: black * 1000,
-      lastUpdate: Date.now()
+      lastUpdate: Date.now() + delay * 10
     };
   };
 
   update(data.white, data.black);
 
   function tick(ctrl, color) {
     var now = Date.now();
-    var millis = times[color] -= now - times.lastUpdate;
-    times.lastUpdate = now;
+    if (now > times.lastUpdate) {
```

Comment:
this is dirty, but I'm not sure of a better way to do it with the current clock design